### ISSUE TITLE
A Fast Adaptive Online Gradient Descent Algorithm in Over-Parameterized Neural Networks

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#327]
### 論文リンク
https://link.springer.com/article/10.1007/s11063-022-11057-4

## 概要
近年、ディープラーニングは多くの実用的なアプリケーションで飛躍的に状態を向上させている。
しかし、この実用性は、学習率、バッチサイズ、ネットワーク初期化などのハイパーパラメータの微調整に大きく依存している。

学習率を調整する1次適応勾配アルゴリズム（Adam、AdaGradなど）が多く提案されているが、特に動的オンライン設定において、過パラメータモデルの学習における初期学習率やネットワーク構造に対して脆弱である。

そのため、深層学習を実際に利用する際には、ハイパーパラメータのチューニングコストをいかに削減するかが大きな課題となっています。
この問題に対処するため、我々はRadhakrishnanらの適応戦略とGhadimiらの加速戦略を統合し、高速適応型オンライン勾配アルゴリズムであるFAOGDを提案する。
適応戦略は過去の勾配と学習損失値に応じて学習速度を調整するのみであり、加速戦略はディープモデルの学習を加速するために用いられるヘビーボールモメンタムである。提案するFAOGDは、学習率に関連するハイパーパラメータを調整する必要がないため、不要な計算オーバーヘッドを大幅に削減できるというメリットを享受することができます。
また、経験的学習率を用いたAdamやAdaGradに匹敵する𝑂(𝑁‾‾√)という後悔限界を得ることができることも示される。オーバーパラメータ化されたニューラルネットワークにおけるシミュレーションの結果、FAOGDが既存のアルゴリズムよりも優れていることが明らかになった。さらに、FAOGDはネットワーク構造やバッチサイズに対してもロバストである。
